right_view_BFS.py

```diff
@@ -0,0 +1,35 @@
+# Time Complexity : Add - O(n)
+# Space Complexity :O(2^h)
```

Comment:
in BFS you'll be traversing according to level, explain the what the variable H is here

Reply:
Yeah, the maximum number of nodes in the queue would be the width(or leaf nodes) which is proportional to 2^h. By h I meant to imply the number of levels.